sql/item_basic.sql

```diff
@@ -4260,9 +4265,9 @@ INSERT INTO `item_basic` VALUES (4525,0,'pumpkin_pie_+1','pumpkin_pie_+1',12,154
 INSERT INTO `item_basic` VALUES (4526,0,'silkworm_egg','silkworm_egg',12,26204,99,0,250);
 INSERT INTO `item_basic` VALUES (4527,0,'jug_of_marys_milk','marys_milk',12,1548,59,0,131);
 INSERT INTO `item_basic` VALUES (4528,0,'crystal_bass','crystal_bass',12,5644,51,1,0);
-INSERT INTO `item_basic` VALUES (4529,0,'rolanberry_(881_ce)','rolanberry_881',12,1548,59,0,24);
-INSERT INTO `item_basic` VALUES (4530,0,'rolanberry_(874_ce)','rolanberry_874',12,1548,59,0,24);
-INSERT INTO `item_basic` VALUES (4531,0,'rolanberry_(864_ce)','rolanberry_864',12,1548,59,0,24);
+INSERT INTO `item_basic` VALUES (4529,0,'rolanberry_(881_c.e.)','rolanberry_881',12,1548,59,0,24);
+INSERT INTO `item_basic` VALUES (4530,0,'rolanberry_(874_c.e.)','rolanberry_874',12,1548,59,0,24);
+INSERT INTO `item_basic` VALUES (4531,0,'rolanberry_(864_c.e.)','rolanberry_864',12,1548,59,0,24);
```

Comment:
dont put punctuation in item names. we strip that for a reason

Reply:
it's weird because other items have it, so i decided to leave tho, let me strip all

Reply:
question what about items like this?

```
-INSERT INTO `item_basic` VALUES (9127,0,'cyllenian_abjuration_hands','cy.abjuration_hn.',1,61520,99,0,0);
-INSERT INTO `item_basic` VALUES (9128,0,'cyllenian_abjuration_legs','cy.abjuration_lg.',1,61520,99,0,0);
-INSERT INTO `item_basic` VALUES (9129,0,'cyllenian_abjuration_feet','cy.abjuration_ft.',1,61520,99,0,0);
```

that become 

```
+INSERT INTO `item_basic` VALUES (9127,0,'cyllenian_abjuration_hands','cyabjuration_hn',1,61520,99,0,0);
+INSERT INTO `item_basic` VALUES (9128,0,'cyllenian_abjuration_legs','cyabjuration_lg',1,61520,99,0,0);
+INSERT INTO `item_basic` VALUES (9129,0,'cyllenian_abjuration_feet','cyabjuration_ft',1,61520,99,0,0);
```

without the punctuation